WooCommerce/WooCommerceTests/Views/SummaryTableViewCellTests.swift

```diff
@@ -81,3 +82,155 @@ final class SummaryTableViewCellTests: XCTestCase {
         XCTAssertEqual(cellStatusLabel?.layer.cornerRadius, mockLabel.layer.cornerRadius)
     }
 }
+
+private extension SummaryTableViewCellTests {
```

Comment:
These mock Objects look like might be useful for testing other objects, not only this cell, would it make sense to make them available (i.e. maybe an internal struct or a class with static methods?)?

Reply:
Maybe? Usually we're testing specific parts of an order or order details, so the JSON files and  mocked object properties vary quite a bit.